requirements.txt

```diff
@@ -5,4 +5,5 @@ antlr4-python3-runtime==4.7
 boto3>=1.9.76
 xmltodict>=0.11.0
 pandas>=0.25.0
+ibmcloudsql
```

Comment:
can you set version number for this package?

Reply:
I set it to the current version.